CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Fix #4383: bump snakeyaml from 1.30 to 1.31
 
 #### New Features
+* Feat: add annotation @PreserveUnknownFields for generation field
```

Comment:
can you please expand a little this sentence, for example saying that it affects the `crd-generator`